pytensor/tensor/extra_ops.py

```diff
@@ -1757,7 +1616,19 @@ def broadcast_arrays(*args: TensorVariable) -> Tuple[TensorVariable, ...]:
         The arrays to broadcast.
 
     """
-    return tuple(broadcast_to(a, broadcast_shape(*args)) for a in args)
+
+    def broadcast_with_others(a, others):
```

Comment:
We discussed with @aseyboldt that it may make sense to generalize `Second` so that it accepts arbitrary many inputs and returns every variable as output. This would become a flat `broadcast_arrays` once `Elemwised`, and make rewrites easier to read. By overriding the `__str__` we can also make it much more readable in `debug_print` than the current nested `Second`